src/components/containers/ContainersListContent.js

```diff
@@ -62,8 +67,22 @@ const ContainersListContent = ({
     // Must be wrapped; effects cannot return promises
     listTemplateActions();
   }, []);
+
+  const listExport = () =>
+    withToken(token, api.containers.listExport)().then(res => res.data)
+
   return <>
-    <AppPageHeader title="Containers" extra={actionsToButtonList("/containers", actions)} />
+    <AppPageHeader
+      title="Containers"
+      extra={[
+        <ExportButton
+          exportFunction={listExport}
+          fileName={'containers'}
```

Comment:
File name is usually written as `filename` in nodejs code, not `fileName`, and  `utils/download.js` uses that convention, as well. As much as possible, keep the same spelling/casing for variables with the same name.

However, I find filename slightly wrong for this prop name because it's not the filename, it's part of what will be used as the filename.

Last thing, for string properties, the normal syntax is `fileName='containers'`, not `fileName={'containers'}`.